functions/controllers/user/imageUpload.js

```diff
@@ -0,0 +1,54 @@
+const Busboy = require('busboy')
+const path = require('path')
+const os = require('os');
+const fs = require('fs')
+const config = require('../../util/config')
+const { admin } = require('../../util/admin')
+const { status, message } = require("../../util/constants");
+var HttpStatus = require("http-status-codes");
```

Comment:
Why are you using var instead of a const here
```suggestion
const HttpStatus = require("http-status-codes");
```

Reply:
the var on the code is copy and paste from the documentaion will change that